modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+
+Multitenancy::
+A software architecture where a single software instance serves multiple yet distinct user groups.
+
+Namespace-scoped instance (Application delivery instance)::
+A mode in which Argo CD is configured to only manage resources in certain namespaces on a cluster and use them for application delivery.
+
+Notifications Controller (Argo CD Notifications Controller)::
+A controller that continuously monitors Argo CD applications and provides a flexible way to notify users about important changes in the application state. 
+
+Progressive delivery::
+In the {gitops-shortname} context, progressive delivery is a process of releasing application updates in a controlled and gradual manner.
+
+{gitops-title}::
+ An Operator that uses Argo CD as the declarative {gitops-shortname} engine to enable {gitops-shortname} workflows across multicluster OpenShift and Kubernetes infrastructure.
+
+Refresh::
+The process to compare the latest code in the Git repository with the live state and determine the difference. For example, in the Argo CD UI, when you click *Refresh*, Argo CD connects to an application's target Git repository, retrieves the content, and then generates manifests from that content. Argo CD then compares that target state against the live cluster state.
+
+Remote cluster::
+A cluster that you can add to Argo CD either declaratively or by using the {gitops-shortname} CLI. Remote cluster is distinct from the local cluster where Argo CD is installed.
+
+Resource Exclusion::
+A configuration you use to exclude resources from discovery and sync so that Argo CD is unaware of them.
+
+Resource Inclusion::
+A configuration you use to include resources to discover, sync, and restrict the list of managed resources globally.
+
+Single tenancy::
+A software architecture where a single software instance serves a single user or group of users.
+
+Sync::
+The process of synchronizing the live state of an application's cluster resources with the target state defined within the Git repository to ensure consistency. Examples include syncing an application by applying changes to a cluster by using the Argo CD UI.
+
+Sync status::
+The status of an application that indicates whether the live state matches the target state.
+
+Target state::
+The wanted state of application resources, as represented by files in a Git repository.
+
+User-defined Argo CD instance::
+A custom Argo CD instance that you install and deploy to manage cluster configurations or deploy applications. By default, any new user-defined instance has permissions to manage resources only in the namespace where it is deployed.
+
+Workload::
+Any process usually defined within resources such as `Deployment`, `StatefulSet`, `ReplicaSet`, `Job`, or `Pod`, running within a container. Examples include a Spring Boot application, a NodeJS Express application, or a Ruby on Rails application.
```

Comment:
```suggestion
Any process, usually defined within resources such as `Deployment`, `StatefulSet`, `ReplicaSet`, `Job`, or `Pod`object, that is running within a container. Examples include a Spring Boot application, a NodeJS Express application, or a Ruby on Rails application.
```

Reply:
@mburke5678 thanks for the suggestion. Please note that here Pod is a resource, hence retaining the content for Pod as is. However, I am implementing the other two suggestions.